### DISCUSSION TITLE
Adapter Override Warning During Trainer.Predict()

### DISCUSSION THREAD

**Discussion by JosephGatto** [adapter-hub/adapters#671] [Q&A] [answered]
I am noticing whenever I call "trainer.predict" after training an adapter model (I am using BART for seq2seq), I get the following warning: 

`WARNING:adapters.loading:Overwriting existing adapter 'adapter'.`

Is this safe to ignore? All I am doing is running Seq2SeqAdapterTrainer.train(), then immediately calling trainer.predict()

**Comment by calpt** [comment, marked as answer]
Depending on your exact training setup, this can be expected. E.g., if you use `load_best_model_at_end=True` to re-load the best checkpoint after training has completed, the existing adapter will be overwritten.

**Reply by JosephGatto** [reply]
Yes that is exactly what I am doing! So this is correct then and the best adapter is whats being loaded?

**Reply by calpt** [reply]
Yes
